src/contour/Config.cpp

```diff
@@ -102,6 +102,56 @@ namespace
                     "Could not create directory {}. {}", path.parent_path().string(), ec.message()) };
     }
 
+    vector<fs::path> getTermInfoDirs(optional<fs::path> const& appTerminfoDir)
+    {
+        auto locations = vector<fs::path>();
+
+        if (appTerminfoDir.has_value())
+            locations.emplace_back(appTerminfoDir.value().string());
+
+        locations.emplace_back(Process::homeDirectory() / ".terminfo");
+
+        if (auto const* value = getenv("TERMINFO_DIRS"); value && *value)
+            for (auto const dir: crispy::split(string_view(value), ':'))
+                locations.emplace_back(string(dir));
+
+        locations.emplace_back("/usr/share/terminfo");
```

Comment:
not sure this should be hard coded. at least for UNIX installs, we could make use of cmake provided POSIX install-prefix (using GNUInstallDirs or so, in cmake, a variable that should than be preprocessor-defined in the CMakeLists.txt

Reply:
Fom man page of `tic` it seems that this is default install path for termifo files, and dont think that it will be changed by any distro

Reply:
we should at least ifdef out WIN32 then 🤔

Reply:
See  getdefaultTERM, we just return "contour" without checking any files